Sources/NIOKit/Extensions/FutureOperators.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  FutureOperators.swift
+//  NIOKit
+//
+//  Created by Jari (LotU) on 06/11/2018.
+//
+
+import NIO
+
+// MARK: - +
```

Comment:
what's the `-` mean in all these MARKs ?

Reply:
It's to make seperators in Xcode